server/public/styles.css

```diff
@@ -0,0 +1,5 @@
+.bg-image {
+  background-image: url("http://cdn.shopify.com/s/files/1/1381/2647/articles/stages_of_weight_loss_1024x1024.png?v=1625161628");
```

Comment:
I recommend downloading this to your server/public so you don't have to worry about link rot.